tests/matchers.py

```diff
@@ -0,0 +1,59 @@
+"""
+Matchers for use in test assertions.
+
+This module contains objects which can be used to write better test assertions.
+Each matcher class defines an __eq__ method, allowing instances of these
+matchers to be used in comparisons wherever you would usually pass a plain
+value.
+
+For example imagine you want to assert that a view returns an HTTP 302 redirect
+to example.com. Without matchers you would have to do something like this:
+
+    result = my_view()
+
+    assert isinstance(result, HTTPFound)
+    assert result.location == "https://example.com"
+
+By using the `Redirect302To` matcher you can simply write:
+
+    assert result == Redirect302To("https://example.com")
```

Comment:
Where matchers _really_ shine is when comparing collections or in mock methods like `assert_called_once_with()`, but we don't yet have any matcher classes here that'd be useful in those ways so I can't use that example.